lib/ramble/ramble/test/end_to_end.py

```diff
@@ -534,3 +535,87 @@ def test_env_var_builtin(mutable_config, mutable_mock_workspace_path, mock_appli
                 if export_found and cmd3_regex.search(line):
                     cmd_found = True
             assert cmd_found and export_found
+
+
+def test_sitevar_dry_run(mutable_config, mutable_mock_workspace_path):
```

Comment:
Does it make sense to test multiple scopes here?

Reply:
I thought about this and I decided not. I like the idea in general but it wasn't clear to me that we actually wanted to enable variables to come in from every possible scope or just site? I was taking the scope of this to be site

Reply:
I think limiting it to 'site' makes for a kind of awkward user experience. i.e. this works:
`ramble config --scope=site add 'config:variables:test_var:"test"'`

but this doesn't:
`ramble config add 'config:variables:test_var:"test"'`

neither does:
`ramble config --scope=defaults add 'config:variables:test_var:"test"'`

And I'm pretty sure if we just remove the scope specification on these lines it'll "just work(TM)".

In lieu of that, I think we need some clear user facing documentation that this is only expected to work within the site scope. If there is a technical reason for this to only be at the site scope though, I think adding the documentation is sufficient. However, I think it would be helpful to know that as well.

Reply:
Awesome, thanks!